osu.Game/Rulesets/Edit/EditorTimestampParser.cs

```diff
@@ -32,7 +33,10 @@ public static class EditorTimestampParser
         /// <item>1:02:300 (1,2,3) - parses to 01:02:300 with selection</item>
         /// </list>
         /// </example>
-        private static readonly Regex time_regex_lenient = new Regex(@"^(((?<minutes>\d{1,3}):(?<seconds>([0-5]?\d))([:.](?<milliseconds>\d{0,3}))?)(?<selection>\s\([^)]+\))?)$", RegexOptions.Compiled);
+        private static readonly Regex time_regex_lenient = new Regex(
+            @"^(((?<minutes>\d{1,3}):(?<seconds>([0-5]?\d))([:.](?<milliseconds>\d{0,3}))?)(?<selection>\s\([^)]+\))?)(?<suffix>\s-.*)?$",
```

Comment:
i dunno about hard-coupling to the ` - blahblahblah` syntax here, if we're doing this we may as well allow almost any characters there as long as they follow a word boundary, but sure, i guess we'll see if someone complains